superset/db_engine_specs/mssql.py

```diff
@@ -87,6 +87,11 @@ class MssqlEngineSpec(BaseEngineSpec):
             SMALLDATETIME(),
             GenericDataType.TEMPORAL,
         ),
+        (
+            re.compile(r"^uniqueidentifier.*", re.IGNORECASE),
+            types.UUID(),
+            GenericDataType.STRING
+        ),
```

Comment:
It would be nice to add a unit test for this to protect against regressions. See `tests/unit_tests/db_engine_specs/test_mssql.py`

Reply:
added unittest